boards/arm/mimxrt1050_evk/mimxrt1050_evk_defconfig

```diff
@@ -15,3 +15,5 @@ CONFIG_CORTEX_M_SYSTICK=y
 CONFIG_GPIO=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=600000000
 CONFIG_ARM_MPU=y
+CONFIG_HEAP_MEM_POOL_SIZE=300000
+CONFIG_CHARACTER_FRAMEBUFFER_DISPLAY_DEV_NAME="ELCDIF_1"
```

Comment:
Please move these to `boards/arm/mimxrt1050_evk/Kconfig.defconfig`

Reply:
ok.